docs/analysis/criteria.md

```diff
@@ -232,16 +247,23 @@ Most CNCF websites are accessed from mobile and other non-desktop devices at
 least 10-20% of the time. Planning for this early in your website's design will
 be much less effort than retrofitting a desktop-first design.
 
+We evaluate on the following:
+
 - Is the website usable from mobile?
 - Are doc pages readable?
-- Are all / most website features accessible from mobile -- such as the top-nav,
-  site search and in-page table of contents?
+- Are all or most website features accessible from mobile -- such as the
+  top-nav, site search, and in-page table of contents?
 - Might a [mobile-first] design make sense for your project?
+- Are color contrasts significant enough for color-impaired readers?
+- Are most website features usable using a keyboard only?
+- Does text-to-speech offer listeners a good experience?
 
 [mobile-first]:
   https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Responsive/Mobile_first
 
-Plan for suitable [accessibility][] measures for your website. For example:
+Plan for suitable [accessibility] measures for your website. For example:
+
+We evaluate on the following:
```

Comment:
Copy-paste error here? It doesn't make sense to have two phrases ending in colons. Maybe you meant:

```suggestion
Plan for suitable [accessibility][] measures for your website. For example, we
evaluate on the following:
```